src/components/picker/searcher.types.ts

```diff
@@ -1,3 +1,11 @@
 import { ListItem } from '../list/list-item.types';
 
+/**
+ * A search function that takes a search-string as an argument, and returns
+ * a promise that will eventually be resolved with an array of `ListItem`:s.
+ *
+ * @param {string} query A search query. Typically what the user has written
+ * in the input field of a limel-picker.
+ * @returns {Promise<ListItem[]>} The search result.
```

Comment:
Unfortunately the _types_ disappear. I'm guessing that's a bug, either in lime-elements or in kompendium.

I'm not sure if we should add the types to the description as a workaround, or just try to find and fix the bug instead? @jgroth @mjlime

Reply:
I'd love if it was all fixed, but do what you can.